### DISCUSSION TITLE
Which repositories are mine?

### DISCUSSION THREAD

**Discussion by David263** [community/community#42498] [Repositories]
### Select Topic Area

Product Feedback

### Body

The Repositories list simply lists all repos, no matter what kind. It would be more useful for me to see clearly which are my own repos, hosted just by me, and which are clones of repos of other people, created to play around with them. Perhaps you could group them into two subsections? Or indicate which are mine in some obvious way?

**Comment by jfullstackdev** [comment]
you can, by looking at the repo name with your username before it

![image](https://user-images.githubusercontent.com/47092464/209423302-0cb0fbcc-b42d-4ff3-ba3b-37026a0400f3.png)

though there is still no indication whether it's a fork, because the other one is a fork from my Org, the other one is the original repo and the other one is a fork of the original repo on the same Org

it should be just like this 

![image](https://user-images.githubusercontent.com/47092464/209423331-6e1b1bce-0151-4976-bb95-ed32d1426c6f.png)

although it's same view at the Stars tab but not in the actual list

![image](https://user-images.githubusercontent.com/47092464/209423345-5e09700c-38a9-4a53-8ef7-bed14e024702.png)

**Comment by David263** [comment]
Here's a screenshot that shows part of the problem:


Oops, pasting my png screenshot here didn't do anything. I use Firefox and frequently have problems that aren't on Chrome. But Firefox is so much nicer.
